index.bs

```diff
@@ -129,7 +129,7 @@ Example implementations of face detection are e.g. <a href="https://developer.an
 
 ## Barcode Detection API
 
-{{BarcodeDetector}} represents an underlying accelerated platform's component for detection in images of QR codes or barcodes.  It provides a single {{BarcodeDetector/detect()}} operation on an {{ImageBitmapSource}} which result is a Promise.  This method MUST reject this Promise in the cases detailed in [[#image-sources-for-detection]]; otherwise it MAY queue a task using the OS/Platform resources to resolve the Promise with a sequence of {{DetectedBarcode}}s, each one essentially consisting on and delimited by a {{DetectedBarcode/boundingBox}} and a series or {{Point2D}}s, and possibly a |rawValue| decoded {{DOMString}}.
+{{BarcodeDetector}} represents an underlying accelerated platform's component for detection in images of QR codes or barcodes.  It provides a single {{BarcodeDetector/detect()}} operation on an {{ImageBitmapSource}} which result is a Promise.  This method MUST reject this Promise in the cases detailed in [[#image-sources-for-detection]]; otherwise it MAY queue a task using the OS/Platform resources to resolve the Promise with a sequence of {{DetectedBarcode}}s, each one essentially consisting on and delimited by a {{DetectedBarcode/boundingBox}} and a series or {{Point2D}}s, and possibly a {{DetectedBarcode/rawValue}} decoded {{DOMString}}.
 
```

Comment:
The changes themselves are OK, please run
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs  > index.html`
and upload the compiled `.html` file as well.

(Having to compile and upload both is kind of lame, but in low traffic repos like this one, maintaining travis jobs is a bit of an overkill, hence the manual compiling).

Also I suspect #22 is not needed if we continue this one, correct?